### ISSUE TITLE
Direct link to stylesheet doesn't work

### ISSUE THREAD

**Issue by billie-spektor** [microsoft/vscode-edge-devtools#184]
I can't get the 'click to open file' to work. Because there are some issues in VS code stable, I'm testing this in VS code insiders. Is this supposed to work or am I missing something?

![image](https://user-images.githubusercontent.com/69010225/89338048-42ab8280-d69c-11ea-9ef6-7e6fce9bcd18.png)

Hover over styles.css gives the path file:///C:/test/styles.css:1, which seems to be correct. When I click on it, nothing happens. The VS code developer tools shows the following:

![image](https://user-images.githubusercontent.com/69010225/89338362-c2395180-d69c-11ea-8361-84b55ebca64d.png)

Version: 
![image](https://user-images.githubusercontent.com/69010225/89338734-4db2e280-d69d-11ea-85d1-925e3094b99c.png)

**Comment by vidorteg** [comment]
Hi @billie-spektor, 

Thanks for reporting this issue and sorry for the  break on Stable channel, we are waiting for the fix to land :).
Back to topic, you are right it should work when pathMapping is set. Did you set the pathMapping? if you did could you paste the configuration? (please remove any personal information before sharing it)

PathMapping:
`pathMapping: This property takes a mapping of URL paths to local paths, to give you more flexibility in how URLs are resolved to local files. "webRoot": "${workspaceFolder}" is just shorthand for a pathMapping like { "/": "${workspaceFolder}" }.`

You can also find more information in this fixed issue description:
https://github.com/microsoft/vscode-edge-devtools/issues/100

If this still does not solve your issues please let us know and we can investigate further 😀

**Comment by billie-spektor** [comment]
Hi,

I've tried a few different options, but they don't seem to have an effect. Based on #100 and the absolute url in my case, I would think I need an empty path mapping. 

![image](https://user-images.githubusercontent.com/69010225/89493687-015ac600-d7b5-11ea-846b-62a4ee6b4cf3.png)

I also tried:
```

    "vscode-edge-devtools.pathMapping": {
        "/": "${workspaceFolder}"
    }
```

In both cases, nothing happens when clicking on the stylesheet link.

Hover over styles.css in the vs code sidebar gives: C:\test\styles.css
Hover over styles.css:1 in the extension gives file:///C:/test/styles.css:1

Thanks for your time!

**Comment by vidorteg** [comment]
Hi @billie-spektor 
Just wanted to let you know that I'm investigating this issue, I'll follow up with some details soon :)

**Comment by vidorteg** [comment]
Thanks for the report @billie-spektor ,I verified and this is indeed a bug. I created a patch to fix this and should be released soon. Thanks for making this project better by reporting issues. 🥇

**Comment by billie-spektor** [comment]
HI @vidorteg . I got this working now in the newest release for local files. thanks for fixing!

For a page served from localhost, I got it partly working. It works well when using a pathmapping in settings.json like this:

```
"vscode-edge-devtools.pathMapping": {
        "/": "C:\\testproject"
    }
```

However, that means that I have to change the settings.json for every project. How can I make this more general by using something like the workspacefolder variable? The direct link to the stylesheet stops working when using this:

```
    "vscode-edge-devtools.pathMapping": {
        "/": "${workspaceFolder}"
    }
```

Should this work? I was expecting the two variants to be equal when the open folder is C:\testproject.

See version and tooltip on hover below:

![image](https://user-images.githubusercontent.com/69010225/94968734-bb468800-0501-11eb-8b7f-3990e6dd7252.png)

![image](https://user-images.githubusercontent.com/69010225/94968868-f779e880-0501-11eb-898c-3dd3c7774c9d.png)
